web/src/admin.html

```diff
@@ -31,9 +31,19 @@
     <div class="container">
       <div class="row">
         <div class="col-md-6 col-md-offset-3 text-center">
-          <button id="seedButton" type="button" class="btn btn-primary">Seed players</button>
+          <button id="seed-button" type="button" class="btn btn-primary">Seed players</button>
         </div>
       </div>
+      <div class="row">
+        <div class="col-md-6 col-md-offset-3">
+          <label>Ranked duration(seconds)</label>
```

Comment:
Nitpick: could we use minutes on the frontend? Just for the convenience of the organiser (we can still use seconds at the API level though)